### PR TITLE
fontFamily

### PR DESCRIPTION
**Pull request by mkempa-np**
added "fontFamilyKey" to typography.kt and set usage in fontFamily property

### CODE REVIEW

File: styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt [hunk 1]
```diff
@@ -133,7 +133,7 @@ interface Typo : StyleParams {
      * @param value extension function parameter for small media devices, recommended to use
      *           predefined values via [dev.fritz2.styling.theme.Theme.fonts] that offer the properties of [Fonts]
      */
-    fun fontFamily(value: Fonts.() -> Property) = property(fontSizeKey, Theme().fonts, value)
+    fun fontFamily(value: Fonts.() -> Property) = property(fontFamilyKey, Theme().fonts, value)
```
Comment (chausknecht) [inline styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt:136]: Please correct the API doc example call too! ``body`` seems not to a an appropriate value there!

File: styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt [hunk 2]
```diff
@@ -162,7 +162,7 @@ interface Typo : StyleParams {
         lg: (Fonts.() -> Property)? = null,
         xl: (Fonts.() -> Property)? = null
     ) =
-        property(fontSizeKey, Theme().fonts, sm, md, lg, xl)
+        property(fontFamilyKey, Theme().fonts, sm, md, lg, xl)
```
Comment (chausknecht) [inline styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt:165]: Please correct the API doc example call too! ``body`` seems not to a an appropriate value there!

**Review by chausknecht — CHANGES_REQUESTED** [review]
Please apply requested changes.

File: styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt (hunk 1 above)
Comment (mkempa-np) [reply styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt:136]: no correction needed

File: styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt (hunk 2 above)
Comment (mkempa-np) [reply styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt:165]: no correction needed

**Review by chausknecht — APPROVED** [review]

**Pull request merged**